packages/frontend-shared/src/components/Slideshow.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="relative">
+    <!-- leave-to classes are marked absolute to allow the entering component to
+    animate on top of the leaving component, creating a fading animation with no thrashing -->
+    <transition
+      enter-from-class="opacity-0"
+      enter-active-class="transition duration-300 ease-out"
+      enter-to-class="opacity-100"
+      leave-from-class="opacity-100 absolute"
+      leave-active-class="transition duration-300 ease-in absolute"
+      leave-to-class="opacity-0 absolute"
```

Comment:
The tailwind + `<transition>` combo sure is nice!

Reply:
I thought we already have a slideshow abstraction somewhere - we definitely have other slideshows in the app. 

Edit... this is the slideshow, it got moved from app to frontend shared? What was the reason for that?

Reply:
Correct, `Promo` is just a thin presentational layer atop the existing `Slideshow` component. At the beginning I had thought this might be potentially useful as a "base" component (eventually used in launchpad, maybe) so I moved it, but that's aspirational thinking so I'll shift this back into `app`